scripts/get-default-key-address.sh

```diff
@@ -1,10 +1,21 @@
 #!/bin/sh
 
+if [ -e configuration/GenesisFiles/GENHASH ]
+then
+    :
+else
+    echo "configuration/GenesisFiles/GENHASH does not exist"
+	/bin/bash ./scripts/genesis.sh
```

Comment:
/bin/bash does not exist on every platform, that's why this script is launched via /usr/bin/env bash
isn't the script/genesis.sh executable?